verifier.sh

```diff
@@ -446,18 +464,20 @@ set -e
 if [ \$GEM_SET_DEBUG ]; then
     set -x
 fi
-echo -e \"y\ny\ny\n\" | oq-platform/openquakeplatform/bin/deploy.sh --host localhost
+echo -e \"y\ny\ny\n\" | oq-platform/openquakeplatform/bin/deploy.sh --host oq-platform.loc.lan
 
 cd oq-platform/openquakeplatform
+
+# add a simulated qgis uploaded layer
+./openquakeplatform/bin/simqgis-layer-up.sh --sitename "http://oq-platform.loc.lan"
+
 export PYTHONPATH=\$(pwd)
-sed 's@^pla_basepath *= *\"http://localhost:8000\"@pla_basepath = \"http://localhost\"@g' openquakeplatform/test/config.py.tmpl > openquakeplatform/test/config.py
+sed 's@^pla_basepath *= *\"http://localhost:8000\"@pla_basepath = \"http://oq-platform.loc.lan\"@g' openquakeplatform/test/config.py.tmpl > openquakeplatform/test/config.py
```

Comment:
I would check before that `oq-platform.loc.lan` is actually resolvable and if not, die; without this check the job will fail but with errors probably hard to be linked to a missing host in `/etc/hosts`.

It can be done simply with
```bash
getent hosts oq-platform.loc.lan > /dev/null
```
if the host does not exists the exit code is `2`